src/aws-manager.js

```diff
@@ -139,10 +125,172 @@ class AwsManager {
     this.reportAmiUsage = series.amiUsage.reporter(influx);
   }
 
+  async init(){
+    try {
+      let rawJson = await this.spotRequestContainer.read('internal-provisioner-data');
+      let data = JSON.parse(rawJson);
+      // This is mainly for debugging
+      fs.writeFileSync('internal-data.json', data);
```

Comment:
hmm.  i guess it really was only there for development.  I could wrap it in an `if (process.env.NODE_ENV !== 'production')`, but i'm leaning towards just putting it in a log.trace.  I don't want to pass the profile into the actual aws-manager.js file because I think that's making it easier to do bad things based on environment, when we should have all that as config stuff....

The more i think about it, log.trace is likely what i'll do.